src/app/App.js

```diff
@@ -0,0 +1,34 @@
+import './App.css';
+import { Counter} from "../shared/ui/Counter/Counter";
+import { GenreSelect } from "../shared/ui/GenreSelect/GenreSelect";
+import { SearchForm } from "../shared/ui/SearchForm/SearchForm";
+import {generList} from "../shared/ui/GenreSelect/GenerMockData";
+
+
+function App() {
+  const geners=['Action', 'Comedy', 'Drama', 'Fantasy'];
+
+  const handleSearch = (val) => {
+    console.log(val);
+  };
+
+  const handleChangeSelectedGenre = (val) => {
+    console.log(val);
```

Comment:
let's delete `console.log`, it can be dangerous from a security perspective, you can put an empty function for now, later will fill this method